schematic/manifest/generator.py

```diff
@@ -1390,19 +1387,19 @@ def get_manifest_with_annotations(
         self.additional_metadata = annotations_dict
 
         # Generate empty manifest using `additional_metadata`
-        manifest_url = self.get_empty_manifest(sheet_url=sheet_url, strict=strict)
+        # With annotations added, regenerate empty manifest
+        manifest_url = self.get_empty_manifest(sheet_url=True, strict=strict)
```

Comment:
Later, `set_dataframe_by_url` function uses manifest_url as an input parameter, so I think here manifest_url has to be an actual google sheet url. I then hard-coded sheet_url=True